plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ScanQueryPageSource.java

```diff
@@ -309,9 +323,11 @@ private Decoder createDecoder(String path, Type type)
         }
         if (type instanceof RowType) {
             RowType rowType = (RowType) type;
+            Map<String, ElasticsearchColumnHandleProperty> children = handleProperty.getChildren();
 
             List<Decoder> decoders = rowType.getFields().stream()
-                    .map(field -> createDecoder(appendPath(path, field.getName().get()), field.getType()))
+                    .map(field -> createDecoder(appendPath(path, field.getName().get()), field.getType(),
```

Comment:
nit: keep all arguments to createDecoder on the same line or different lines consistently